freqtrade/strategy/interface.py

```diff
@@ -19,30 +20,80 @@ class IStrategy(ABC):
         ticker_interval -> str: value of the ticker interval to use for the strategy
     """
 
+    # associated minimal roi
     minimal_roi: Dict
+
+    # associated stoploss
     stoploss: float
+
+    # associated ticker interval
     ticker_interval: str
 
-    @abstractmethod
+    # configuration used, just in case the strategy want's to use it for something
+    config: dict = {}
```

Comment:
Do you have any use case how the config could be used here? I'm not sure if we should add it without knowing what we need from that.